PSix/FacebookEventParser.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  FacebookEventParser.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/20/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+
+class FacebookEventsParser {
```

Comment:
I would make this a `struct` with a static method which returns `Events`, but up to you.